_includes/header.html

```diff
@@ -1,217 +1,181 @@
+                  <a href="https://algorithmia.com/blog" class="syn-ph-48 syn-pv-16">
+                    <div>
+                      <div class="syn-font-weight-bold">Blog</div>
+                      <div class="link-description">
+                        Special topics in machine learning
+                      </div>
+                    </div>
+                  </a>
+                </div>
+                <div class="syn-dropdown-list-item syn-body-2 syn-mb-0">
+                  <a href="https://algorithmia.com/resources" class="syn-ph-48 syn-pv-16">
+                    <div>
+                      <div class="syn-font-weight-bold">Resources</div>
+                      <div class="link-description">
+                        Case studies, videos, and reports
+                      </div>
+                    </div>
+                  </a>
+                </div>
+                <div class="syn-dropdown-list-item syn-body-2 syn-mb-0">
+                  <a href="https://algorithmia.com/developers" class="syn-ph-48 syn-pv-16">
+                    <div>
+                      <div class="syn-font-weight-bold">Docs</div>
+                      <div class="link-description">
+                        Platform technical documentation
+                      </div>
+                    </div>
+                  </a>
+                </div>
+                <div class="syn-dropdown-list-item syn-body-2 syn-mb-0">
+                  <a href="https://algorithmia.com/events" class="syn-ph-48 syn-pv-16">
+                    <div>
+                      <div class="syn-font-weight-bold">Events</div>
+                      <div class="link-description">
+                        Webinars, talks, and trade shows
+                      </div>
+                    </div>
+                  </a>
+                </div>
         </div>
-        {% endunless %}
-        <a
-          href="/signin"
-          class="show-unauth syn-link-no-decoration"
+      </div>
+      <a
+          href="https://algorithmia.com/demo"
+          class="syn-btn text syn-ml-8 syn-link-no-decoration display-inline-flex"
         >
-          <button class="syn-btn text theme-primary syn-visible-lg syn-ml-16">
-            Sign in
-          </button>
-        </a>
-        <a href="/signup" class="show-unauth syn-link-no-decoration">
-          <button
-            class="syn-btn outlined theme-primary syn-visible-lg syn-ml-16"
-          >
-            {% if site.enterprise %}
-            <span>Create Account</span>
-            {% else %}
-            <span>Try It For Free</span>
-            {% endif %}
-          </button>
+          Demo
         </a>
-        {% unless site.enterprise %}
-        <a href="/demo" class="show-unauth syn-link-no-decoration">
-          <button
-            class="syn-btn contained theme-accent syn-visible-lg syn-ml-16"
-          >
-            Get Your Demo
-          </button>
+        <a href="https://teams.algorithmia.com/signup"
```

Comment:
Same comment here about keeping `/signup`
```suggestion
        <a href="/signup"
```